src/test/ui/parser/item-free-static-no-body-semantic-fail.rs

```diff
@@ -3,9 +3,9 @@
 fn main() {}
 
 static A: u8; //~ ERROR free static item without body
-static B; //~ ERROR free static item without body
+static B;
 //~^ ERROR missing type for `static` item
 
 static mut C: u8; //~ ERROR free static item without body
-static mut D; //~ ERROR free static item without body
```

Comment:
I think you removed the wrong error label

Reply:
Thanks, I have addressed this comment

Reply:
It still looks to me like you've removed the "free static item without body" label.